README.md

```diff
@@ -15,6 +15,56 @@ Namely:
 
 Run `make install`
 
+## How to use custom CSS stylesheet
```

Comment:
I actually think this section deserves its own document.

Could you move this entire section to `custom_css.md` and link to it from the README?

Reply:
Sure!